dist/template/package.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "YOUR-APP-NAME",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "scripts": {
+        "kakunin": "cross-env NODE_ENV=prod kakunin"
+    },
+    "author": "",
+    "license": "ISC",
+    "dependencies": {
+        "cross-env": "^5.2.0",
+        "kakunin": "2.5.0"
```

Comment:
Old version of Kakunin. TBH it should get the newest available version dynamically and put it there. Otherwise we would always use the old version.

What's more I'd like to have an option to change kakunin version to the one of my choice

Reply:
Changed in template `package.json` into `"kakunin": "latest"`

Right now you can provide the version that you need. e.g `kakunin-cli cp /example/path 2.3.0` after that you will use 2.3.0 version.